rskj-core/src/test/java/co/rsk/peg/feeperkb/constants/FeePerKbConstantsTest.java

```diff
@@ -0,0 +1,98 @@
+package co.rsk.peg.feeperkb.constants;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FeePerKbConstantsTest {
+
+    @ParameterizedTest
+    @MethodSource("getGenesisFeePerKbTestProvider")
+    void getGenesisFeePerKbTest(FeePerKbConstants feePerKbConstants, Coin expectedGenesisFeePerKb) {
```

Comment:
I wouldn't call all each test function `xxxTest`. It's already implied that it's a test

Reply:
I just removed the `Test` `suffix` on methods names to accomplish RootStock standards.